ocaml/xapi/message_forwarding.ml

```diff
@@ -1053,6 +1053,16 @@ functor
 
       let wait_for_tasks = Helpers.Task.wait_for
 
+      let create_vm_message ~__context ~body ~message ~vm =
```

Comment:
```suggestion
      let create_vm_message ~__context ~vm ~message_body ~message =
```
Changing the order and the name of the content parameter will make call sites look more straight-forward:

```ocaml
        create_vm_message ~__context ~vm ~message_body
          ~message:Api_messages.vm_checkpointed;
```